Cue/app/api/Library.js

```diff
@@ -75,7 +75,25 @@ var LibraryApi = {
 			});
 		console.info('Adding deck "' + uuid + '" to library from endpoint: ' + endpoint)
 		return CueApi.fetch(endpoint, method, body);
-	}
+	},
+
+  overwriteDeck(deck: Deck) {
+    let endpoint = '/api/v1/deck/' + deck.uuid
+    let method = 'POST'
+    let cards = deck.cards.map(card => {return {
+      ...card,
+      uuid: card.uuid ? card.uuid : null
+    }})
+    let body = JSON.stringify({
+      name: deck.name,
+      tags: deck.tags,
+      public: Boolean(deck.public), //server returns 400 without Boolean conversion
```

Comment:
the response we get back from sever gives an int for the public property. So if this isn't changed it stays as an int in the decks store object. So when we stringify the int it doesn't give true | false